packages/gui/tests/components/Navigation.test.tsx

```diff
@@ -1,50 +1,67 @@
+jest.mock("../../src/dialogs/InfoDialog", () => () => <div>InfoDialog</div>);
+jest.mock("../../src/dialogs/SettingsDialog", () => () => (
+  <div>SettingsDialog</div>
+));
 
-test('renders component correctly', () => {
-  const { queryByText, getByText, getAllByText, getByPlaceholderText, getByTitle, getByLabelText } = render(
+test("renders component correctly", () => {
+  const {
+    queryByText,
+    getByText,
+    getAllByText,
+    getByPlaceholderText,
+    getByTestId,
+    getByLabelText,
+  } = render(
     <GlobalProvider>
-      <Navigation />
+      <ThemeProvider theme={theme}>
+        <Navigation />
+      </ThemeProvider>
     </GlobalProvider>
   );
-  expect(getByText('some name')).toBeTruthy();
-  expect(getAllByText('ModeSelector')).toHaveLength(2);
-  expect(queryByText('InfoDialog')).not.toBeTruthy();
-  expect(queryByText('John Doe')).not.toBeTruthy();
-  expect(queryByText('SettingsDialog')).not.toBeTruthy();
-  expect(queryByText('ThemeDialog')).not.toBeTruthy();
-  expect(queryByText('InfoDialog')).not.toBeTruthy();
+  expect(getByText("some name")).toBeTruthy();
+  expect(getAllByText("ModeSelector")).toHaveLength(2);
+  expect(queryByText("InfoDialog")).not.toBeTruthy();
+  expect(queryByText("John Doe")).not.toBeTruthy();
+  expect(queryByText("SettingsDialog")).not.toBeTruthy();
+  expect(queryByText("ThemeDialog")).not.toBeTruthy();
+  expect(queryByText("InfoDialog")).not.toBeTruthy();
 
-  userEvent.click(getByText('some name'));
-  const input = getByPlaceholderText('Type...');
-  userEvent.type(input, 'John Doe');
-  userEvent.click(input.parentElement?.querySelector('button')!);
-  expect(getByText('some nameJohn Doe')).toBeTruthy();
+  userEvent.click(getByText("some name"));
+  const input = getByPlaceholderText("Type...");
+  userEvent.type(input, "John Doe");
+  userEvent.click(input.parentElement?.querySelector("button")!);
+  expect(getByText("some nameJohn Doe")).toBeTruthy();
+
+  userEvent.click(getByTestId("navigation-toggle-global-scope"));
 
-  userEvent.click(getByTitle('Toggle Global vs Workspace Scope (Workspace Scope)'));
   expect(setGlobalScope).toBeCalledTimes(1);
 
-  userEvent.click(getByLabelText('Open Settings'));
-  expect(getByText('SettingsDialog')).toBeTruthy();
-  userEvent.click(getByLabelText('Switch Theme'));
-  expect(getByText('ThemeDialog')).toBeTruthy();
-  userEvent.click(getByLabelText('Show Info'));
-  expect(getByText('InfoDialog')).toBeTruthy();
-  userEvent.click(getByLabelText('Send Feedback'));
-  expect(getByText('FeedbackDialog')).toBeTruthy();
+  userEvent.click(getByLabelText("Open Settings"));
+  expect(getByText("SettingsDialog")).toBeTruthy();
+  userEvent.click(getByLabelText("Switch Theme"));
+  expect(getByText("ThemeDialog")).toBeTruthy();
+  userEvent.click(getByLabelText("Show Info"));
+  expect(getByText("InfoDialog")).toBeTruthy();
+  userEvent.click(getByLabelText("Send Feedback"));
+  expect(getByText("FeedbackDialog")).toBeTruthy();
```

Comment:
I think I accidentally triggered prettier here, adding a bunch of quotes. Do we have a formatter which I can use to bring the styling back in line?

Reply:
Nope .. we use `"` and `'` interchangeably in the code base, worth at some point to align this and format the code base